migration/incremental/021-post-restore.sql

```diff
@@ -0,0 +1,72 @@
+
+CREATE FUNCTION public.promscale_post_restore()
+RETURNS void
+SECURITY DEFINER
+SET search_path = pg_catalog
+AS $func$
+DECLARE
+    _sql text;
+BEGIN
+    CREATE TRIGGER ts_insert_blocker
+    BEFORE INSERT ON _ps_trace.span
+    FOR EACH ROW EXECUTE FUNCTION _timescaledb_internal.insert_blocker();
+
+    CREATE TRIGGER ts_insert_blocker
+    BEFORE INSERT ON _ps_trace.event
+    FOR EACH ROW EXECUTE FUNCTION _timescaledb_internal.insert_blocker();
+
+    CREATE TRIGGER ts_insert_blocker
+    BEFORE INSERT ON _ps_trace.link
+    FOR EACH ROW EXECUTE FUNCTION _timescaledb_internal.insert_blocker();
+
+    FOR _sql IN
```

Comment:
let's add a comment explaining why here

Reply:
resolved in [486d8b5](https://github.com/timescale/promscale_extension/pull/185/commits/486d8b5e16f713ac139bbee875c8c801f1d9868c)